ehrapy/plot/_survival_analysis.py

```diff
@@ -293,5 +296,133 @@ def kaplan_meier(
 
     if not show:
         return ax
+
     else:
         return None
+
+
+def coxph_forestplot(
+    coxph: CoxPHFitter,
+    labels: list[str] | None = None,
+    fig_size: tuple = (10, 10),
+    t_adjuster: float = 0.1,
+    ecolor: str = "dimgray",
+    size: int = 3,
+    marker: str = "o",
+    decimal: int = 2,
+    text_size: int = 12,
+    color: str = "k",
+):
+    """Plots a forest plot of the Cox Proportional Hazard model.
+    Inspired by the forest plot in the zEpid package in Python.
+    Link: https://zepid.readthedocs.io/en/latest/Graphics.html#effect-measure-plots
+
+    Args:
+        coxph: Fitted CoxPHFitter object.
+        labels: List of labels for each coefficient, default uses the index of the coxph.summary.
+        fig_size: Width, height in inches.
+        t_adjuster: Adjust the table to the right.
+        ecolor: Color of the error bars.
+        size: Size of the markers.
+        marker: Marker style.
+        decimal: Number of decimal places to display.
+        text_size: Font size of the text.
+        color: Color of the markers.
+
```

Comment:
Having an argument `show` as we do in many plotting functions would be nice (see e.g. [here](https://github.com/theislab/ehrapy/blob/419f2b2f8d9f21d4528ae7c84ce77a48ec48ee31/ehrapy/tools/cohort_tracking/_cohort_tracker.py#L190))